Automattic/pocket-casts-android
# Bookmarks - Add sorting

## Description

This adds sorting to the bookmarks list.

## Testing Instructions

1. Add a few bookmarks to the bookmarks table with different `create_at` and `time` values
2. Launch the app and login with Plus account
3. Open the Full Screen Player
4. Swipe to the bookmarks list
5. Tap the vertical <kbd>...</kbd> button
6. ✅ Verify you see an options menu with 2 options: Select bookmarks, Sort by
7. ✅ Verify the default sort by option is newest to oldest
8. Tap on Select bookmarks
9. ✅ Verify the view enters multi select mode
10. Tap back arrow, then tap the more (...) button again
11. Tap the Sort by option
12. ✅ Verify a new options menu opens with 3 options: "Newest to oldest", "Oldest to newest", and "Timestamp"
13. Tap on the "Oldest to newest"
14. ✅ Verify the list is now sorted with the oldest bookmarks first
15. Open the sort menu again, choose "Timestamp"
16. ✅ Verify the list is now sorted with the earliest timestamp at the top

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/c57a1b1d-30d8-4bc1-bd66-17ba8f3ea1a8

## Checklist
 
#### I have tested any UI changes...
